website/README.md

```diff
@@ -75,7 +83,7 @@ There are several custom markdown plugins that are available by default that enh
 - If you see `# Headline ((#slug))`, this is an example of an [anchor link alias](https://github.com/hashicorp/remark-plugins/tree/je.anchor-link-adjustments/plugins/anchor-links#anchor-link-aliases). It adds an extra permalink to a headline for compatibility and is removed from the output.
 - Due to [automatically generated permalinks](https://github.com/hashicorp/remark-plugins/tree/je.anchor-link-adjustments/plugins/anchor-links#anchor-links), any text changes to _headlines_ or _list items that begin with inline code_ can and will break existing permalinks. Be very cautious when changing either of these two text items.
 
-  Headlines are fairly self-explanatory, but here's an example of how to list items that begin with inline code look.
+  Headlines are fairly self-explanitory, but here's an example of how list items that begin with inline code look.
```

Comment:
Typo here as well

Reply:
Thanks for catching these - added to https://github.com/hashicorp/nextjs-scripts/pull/161 👍